src/main/java/org/betonquest/betonquest/modules/config/ConfigUpdater.java

```diff
@@ -0,0 +1,19 @@
+package org.betonquest.betonquest.modules.config;
+
+import java.io.IOException;
+
+/**
+ * This interface describes something that can update a value of the configuration.
+ *
+ * @param <T> the type of the configuration value
+ */
+public interface ConfigUpdater<T> {
+
+    /**
+     * Update the configuration value.
+     *
+     * @param value the new value
+     * @throws IOException when persisting the changed value fails
+     */
+    void update(T value) throws IOException;
+}
```

Comment:
I see the reason why you created this for the HistoryHandler, but i suggest to delete this interface, and instead use  a Callback for that, because i think first this is not really reusable(or at least will not be reused)  and secondly is this not more or less the same as a callback?